app/src/main/java/com/fs_sournary/weather/utils/ConvertTimeUtils.java

```diff
@@ -0,0 +1,24 @@
+package com.fs_sournary.weather.utils;
+
+import java.text.SimpleDateFormat;
+import java.util.Date;
+
+/**
+ * Created by fs-sournary.
+ * Date on 11/18/17.
+ * Description:
+ */
+
+public class ConvertTimeUtils {
+
+    private SimpleDateFormat mSimpleDateFormat;
+
+    public ConvertTimeUtils() {
+        mSimpleDateFormat = new SimpleDateFormat("MM/dd/yyyy  HH:mm:ss");
```

Comment:
WARNING: [SimpleDateFormat] To get local formatting use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()`, or use `new SimpleDateFormat(String template, Locale locale)` with for example `Locale.US` for ASCII dates.
 Almost all callers should use `getDateInstance()`, `getDateTimeInstance()`, or `getTimeInstance()` to get a ready-made instance of SimpleDateFormat suitable for the user's locale. The main reason you'd create an instance this class directly is because you need to format/parse a specific machine-readable format, in which case you almost certainly want to explicitly ask for US to ensure that you get ASCII digits (rather than, say, Arabic digits).

Therefore, you should either use the form of the SimpleDateFormat constructor where you pass in an explicit locale, such as Locale.US, or use one of the get instance methods, or suppress this error if really know what you are doing.